test/unit/math/mix/fun/trace_gen_inv_quad_form_ldlt_test.cpp

```diff
@@ -64,8 +55,8 @@ TEST(mathMixMatFun, traceGenInvQuadForm) {
 
   // exception tests
   // non-square second arg
-  Eigen::MatrixXd a34(3, 4);
```

Comment:
Can't take an LDLT of a non-square matrix, so this fails before it gets to `trace_gen_inv_quad_form_ldlt` (which takes an LDLT as an argument)